libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -91,10 +91,10 @@ export class CosmosAPI {
   getChainId = async (): Promise<string> => {
     const { data } = await network({
       method: "GET",
-      url: `${this.defaultEndpoint}/node_info`,
+      url: `${this.defaultEndpoint}/cosmos/base/tendermint/v1beta1/node_info`,
```

Comment:
@hedi-edelbloute @henrily-ledger `v1beta1` should be replaced with `{this.version$}`

Reply:
will do in branch